packages/@sanity/mutator/package.json

```diff
@@ -50,14 +50,14 @@
   },
   "dependencies": {
     "@sanity/diff-match-patch": "^3.1.1",
+    "@sanity/types": "3.52.2",
```

Comment:
Having `@sanity/types` as a regular dependency is a pattern already used by `@sanity/migrate`, `@sanity/schema`, `@sanity/util`, and `sanity` itself. And is the preferred way of declaring them.